src/aspire/volume/__init__.py

```diff
@@ -287,6 +358,91 @@ class FBBasisVolume(BasisVolume):
     pass
 
 
+def gaussian_blob_Cn_vols(
+    L=8, C=2, K=16, alpha=1, subscript=1, seed=None, dtype=np.float64
+):
+    """
+    Generate Cn rotationally symmetric volumes composed of Gaussian blobs.
+    The volumes are symmetric about the z-axis.
+
+    Defaults to volumes with no symmetry.
+
+    :param L: The size of the volumes
+    :param C: The number of volumes to generate
+    :param K: The number of blobs each volume is composed of.
+    A Cn symmetric volume will be composed of n times K blobs.
+    :param subscript: The subcript n indicating Cn symmetry.
+    :param alpha: A scale factor of the blob widths
+
+    :return: A Volume instance containing C Gaussian blob volumes with Cn symmetry.
+    """
+
+    def _eval_gaussian_blobs(L, Q, D, mu, subscript, dtype=np.float64):
+        try:
```

Comment:
Maybe a bit overkill. My thinking was that there are molecular symmetry types such as C2h, C2v, C3h, C3v, ... that are not currently supported by this function and would cause an error when trying to convert the string `subscript` (now `order`) to an integer. The idea was to provide a more meaningful error message. `Q` and `D` are parameters that are internal to this method so the user doesn't touch them. If an invalid value for `L` was entered there is already a meaningful error message.

All that said, this is still probably overkill. Especially if we are not concerned with supporting the above mentioned symmetry types.

Reply:
Ok but that check could be moved out into a user-facing function, no?

Reply:
Good point. I've moved this check into `gaussian_blob_vols`.